.github/workflows/create-jira-issues-for-community-activities.yml

```diff
@@ -2,15 +2,16 @@ name: Create Jira issues for community activities
 
 on:
   discussion:
-    types: created
+    types: [created]
   issues:
-    types: opened
+    types: [opened]
   pull_request:
-    types: opened
+    types: [opened]
 
 jobs:
   call-publish-workflow:
```

Comment:
The job name "call-publish-workflow" should be changed to "create-jira-issues-for-community-activities" as it is in the template. And everywhere else where it is not changed.

Reply:
aa now I understand what was meant by job naming ... 
I changed it